### PR TITLE
FakeNewsDetector is working with 92.2% accuracy

### PR DESCRIPTION
**Pull request by rozsnyaitamas**

### CODE REVIEW

File: 01DetectingFakeNews/01DetectingFakeNews.ipynb [hunk 1]
```diff
@@ -0,0 +1,280 @@
...
+    },
+    "vscode": {
+      "interpreter": {
+        "hash": "ffb21c7f50ccbdd9cabeaacb722f478577479649672a627763d22c6e44daa1fc"
+      }
+    }
+  },
+  "nbformat": 4,
+  "nbformat_minor": 0
+}
```
Comment (CzakoZoltan08) [inline 01DetectingFakeNews/01DetectingFakeNews.ipynb]: nice job! as an exercise I would like you to use https://scikit-learn.org/stable/modules/grid_search.html to select the best classification model (it is a more generic solution)
Leave the code as it is and continue with this grid search (so don't alter the existing code, but continue the notebook with a new lines)

Comment (CzakoZoltan08) [reply 01DetectingFakeNews/01DetectingFakeNews.ipynb]: you can also try other automatic AI model selection algorithms, like AutoSklearn, Tpot and others

File: 01DetectingFakeNews/01DetectingFakeNews.ipynb [hunk 2]
```diff
@@ -0,0 +1,280 @@
...
+          "text": [
+            "[[584  54]\n",
+            " [ 44 585]]\n"
+          ]
+        }
+      ],
+      "source": [
+        "from sklearn.metrics import confusion_matrix\n",
+        "cm = confusion_matrix(y_test, y_pred)\n",
+        "print(cm)"
```
Comment (CzakoZoltan08) [inline 01DetectingFakeNews/01DetectingFakeNews.ipynb]: please draw a confusion matrix using matplotlib or plotly

File: 01DetectingFakeNews/01DetectingFakeNews.ipynb [hunk 3]
```diff
@@ -0,0 +1,280 @@
...
+        {
+          "name": "stdout",
+          "output_type": "stream",
+          "text": [
+            "Accuracy: 92.27%\n"
+          ]
+        }
+      ],
+      "source": [
+        "from sklearn.metrics import accuracy_score\n",
```
Comment (CzakoZoltan08) [inline 01DetectingFakeNews/01DetectingFakeNews.ipynb]: the accuracy is great but it can have some drawbacks (for example in case of imbalanced data, see https://developers.google.com/machine-learning/crash-course/classification/accuracy)
please calculate other metrics also, like precision, recall, f1-score https://neptune.ai/blog/performance-metrics-in-machine-learning-complete-guide
